web/docs/tql2/operators/chart_area.md

```diff
@@ -0,0 +1,56 @@
+# chart_area
+
+Chart values on a X-Y area chart.
+
+```tql
+chart_area x=any, y=any, [x_min=any, x_max=any, resolution=duration, x_log=bool, y_log=bool, group=any, position=string]
+```
+
+## Description
+
+Chart values on X-Y area chart.
+
+### `x = any`
+
+The values to plot on the X-axis.
```

Comment:
Maybe it's just me, but this sounds a bit weird. Values are not plotted on the x-axis, right?

Reply:
Maybe? I felt this was simple enough to understand. Can you come up with a more precise one?

Reply:
I like simple. It's just that this sounds factually incorrect to me. Maybe "The position of each point along the x-axis" or something like that?